.github/CODEOWNERS

```diff
@@ -136,12 +159,52 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
+src/applications/static-pages/dependency-verification @department-of-veterans-affairs/benefits-ce-dependents
```

Comment:
Why isn't this recognized as a VA Github team?

Reply:
Maybe the team doesn't have write access to the repo? That would seem strange though